tests/test_moxunit_runtests.m

```diff
@@ -0,0 +1,228 @@
+function test_suite=test_moxunit_runtests
+    initTestSuite;
+
+function test_moxunit_runtests_basics
```

Comment:
I agree it's a bit large - and also it takes a while to run (about 3 seconds).
However it tests all possible combinations for seven boolean variables, which may be nice to have. Alternatively one could maybe limit the number of test cases, but one would lose testing for all combinations. 
Do you have ideas on how to break it down in multiple test cases?